tests/testArgumentParser.py

```diff
@@ -64,7 +64,9 @@ class SampleConfig(pexConfig.Config):
     floatItem = pexConfig.Field(doc="sample float field", dtype=float, default=3.1)
     strItem = pexConfig.Field(doc="sample str field", dtype=str, default="strDefault")
     subItem = pexConfig.ConfigField(doc="sample subfield", dtype=SubConfig)
-    
+    choiceItem = pexConfig.ChoiceField(doc = "sample ChoiceField (# multiline doc)", dtype = str,
```

Comment:
Momentary confusion as this isn't a multi-line string. Of course, it is when the allowed values are appended. Is it worth making it explicitly multi-line (ie, include a `\n`) just to drive home the point (and stop me from double-taking next time...)?